### PR TITLE
Created New Pom class

### PR DESCRIPTION
**Pull request by VeenaSkote**
please review

### CODE REVIEW

File: ComcastCRMGUIFramwork/src/main/java/com/comCast/crm/objectRepository/CreateDashboardPage.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.openqa.selenium.support.PageFactory;
+
+public class CreateDashboardPage
+{
+	public CreateDashboardPage(WebDriver driver)
+	{
+		//this.driver=driver;
+		PageFactory.initElements(driver,this);
+	}
+	@FindBy(name="lastname")
```
Comment (VeenaSkote) [inline ComcastCRMGUIFramwork/src/main/java/com/comCast/crm/objectRepository/CreateDashboardPage.java:15]: change the xpath .do not hard code the data

**Review by VeenaSkote — COMMENTED** [review]
added few comments finish your review

File: ComcastCRMGUIFramwork/src/main/java/com/comCast/crm/objectRepository/CreateDashboardPage.java (hunk 1 above)
Comment (VeenaSkote) [reply ComcastCRMGUIFramwork/src/main/java/com/comCast/crm/objectRepository/CreateDashboardPage.java:15]: fixed

**Comment by Veenakote** [comment]
merged

**Pull request merged**